pcdsdevices/pseudopos.py

```diff
@@ -167,4 +168,23 @@ class SimDelayStage(DelayBase):
 
 class PseudoSingleInterface(PseudoSingle, FltMvInterface):
     """PseudoSingle with FltMvInterface mixed in."""
-    pass
+    notepad_setpoint = Cpt(
+        NotepadLinkedSignal, ':OphydSetpoint',
+        notepad_metadata={'record': 'ao', 'default_value': 0.0},
+    )
+
+    notepad_readback = Cpt(
+        NotepadLinkedSignal, ':OphydReadback',
+        notepad_metadata={'record': 'ai', 'default_value': 0.0},
+    )
+
+    def __init__(self, prefix='', parent=None, **kwargs):
+        if not prefix:
+            # PseudoSingle generally does not get a prefix. Fix that here,
+            # or 'notepad_setpoint' and 'notepad_readback' will have no
+            # prefix.
+            prefix = parent.prefix
```

Comment:
We'll need to add prefixes to the PseudoPositioners with multiple pseudo axes to avoid name conflicts here

Reply:
Good call. Adding on `:{self.attr_name}` would probably be the best option.

Reply:
Resolved in 90b479a1a8